### ISSUE TITLE
Allow to create deactivated Hosts via Self-Service-API

### ISSUE THREAD

**Issue by widhalmt** [Icinga/icingaweb2-module-director#1378]
Hi,

A customer of mine is using staged setups for their windows hosts. They are deploying hosts and using the Icinga 2 Powershell Module to create the host objects in Director via Self Service API. The problem is that the hosts are not fully set up and would produce false alarms when Icinga 2 started monitoring right away after the setup (and deploy). This is mostly due to several reboots which won't be scheduled as downtimes.

Due to the mass of special systems they are using they don't want to use many different Self Service API Keys but use a "catchall" key to add the key only to a very basic template and change the imports when setup is done.

I opened Issue Icinga/icinga2-powershell-module#71 to allow the creation of disabled host objects via the Powershell Module but learned that the Self Service API won't allow adding disabled hosts.

So long story short: Could you add a way to add disabled (in Director) objects via the Self-Service API?

Cheers,
Thomas
